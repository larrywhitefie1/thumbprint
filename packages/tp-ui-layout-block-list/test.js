const sass = require('node-sass');

it('compiles correctly', () => {
    const css = sass
        .renderSync({ file: 'packages/tp-ui-layout-block-list/_index.scss' })
        .css.toString();
    expect(css).toMatchSnapshot();
});
