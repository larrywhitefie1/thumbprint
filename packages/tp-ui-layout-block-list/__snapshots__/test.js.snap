// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compiles correctly 1`] = `
"
.tp-block-list-group,
.tp-block-list {
  list-style: none;
  margin-left: 0; }
  .tp-block-list-group > li,
  .tp-block-list > li {
    padding: 6px 12px;
    border-top: 1px solid #e9eced; }
  
  .tp-block-list__link {
    display: block;
    cursor: pointer;
    padding: 6px 12px;
    margin-top: -6px;
    margin-left: -12px;
    margin-right: -12px;
    margin-bottom: -6px; }
    
    .tp-block-list__link:hover {
      background-color: #fafafa; }
    
    .tp-block-list__link--active {
      background-color: #fafafa; }
  
  .tp-block-list--flush > li {
    padding-left: 0;
    padding-right: 0; }
    
    .tp-block-list--flush > li .tp-block-list__link {
      padding-left: 0;
      padding-right: 0;
      margin-left: 0;
      margin-right: 0; }
  
  .tp-block-list--all > li:last-child {
    border-bottom: 1px solid #e9eced; }
  
  .tp-block-list--no-border > li {
    border-top: 0; }


.tp-block-list-group {
  border: 1px solid #e9eced;
  border-radius: 4px; }
  
  .tp-block-list-group > li:first-child {
    border: 0; }
"
`;
